java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_2/UserAccount.java

```diff
@@ -0,0 +1,23 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class UserAccount{
+    public int userId;
```

Comment:
as per encapsulation, fields should be usually defined with the access modifier `private`